POS.html

```diff
@@ -0,0 +1,224 @@
+	}
+	th{
+		padding-top: 5px;
+    	padding-right: 11px;
+    	padding-bottom: 5px;
+    	padding-left: 11px;
+    	text-align: left;
+	}
+	td{
+    	padding-top: 5px;
+    	padding-bottom: 10px;
+    	padding-left: 2px;
+    	text-align: left;
+	}
+
+
+	#IN{
+		float:left;
+		width:112px;
+	}
+	#I{
+		float:left;
+		width:95px;
+	}
+	#Q{
+		float:left;
+		width:81px;
+	}
+	#P{
+		float:left;
+		width:55px;
+	}
+
+
+	#ID{
+		float:left;
+		width:97px;
+	}
+	#PN{
+		float:left;
+		width:96px;
+	}
+	#Sup{
+		float:left;
+		width:75px;
+	}
+	#Price{
+		float:left;
+		width:46px;
+	}
+
+</style>
+
+
+
+ <body>
+ 
+ <div id="header">
+ 	<h1> Bob's Grocery</h1>
+ </div>
+ 
+
+
+<div id="nav">
+ 	<a href="Managerpage.html"><button type="button">Manager Page</button></a><br><br>
+ 	<a href="/login"><button type="button">Log Out</button></a><br>
+ </div>
+
+
+
+<div id="section">
+
+
+	<form action="/search">
+		<input type="text" name="ItemNumber"></input><br>
+
+
+	  	<select name="items">
+	    	<option value=""></option>
+	    	<option value="price">Item Price</option>
+	   		<option value="id">Product Id</option>
+	   		<option value="name">Product Name</option>
+	   		<option value="supply">Supplier</option>
+	   	</select><br>
+
+		<input type="submit" value="Search">
+	</form><br><br>
+		
+	<div style="height:450px;width:332px;border:1px solid #ccc;overflow:auto;">
+
+		<table id="headerTable" border='1'>
+			<tr>
+				<th>Product ID</th>
+				<th>Item Name</th>
+				<th>Supplier</th>
+				<th>Price</th>
+			</tr>
+		</table>
+
+		<table id="resultTable" border='0'>
+			<tr style="font-size:13px">
+				{% for thing in results %}
+				<td><div id="ID">
+						</div>{{thing[0]}}</td>
+				<td><div id="PN">
+						</div>{{thing[1]}}</td>
+				<td><div id="Sup">
+						</div>{{thing[2]}}</td>
+				<td><div id="Price">
+						</div>{{thing[3}}]</td>
+
+				{% end for %}
+			</tr>
+
+		</table>
+	</div>
+</div>
+
+ 
+ <div id="other">
+ 
+ 
+
+ 
+ 	<div style="height:400px;width:360px;border:1px solid #ccc;overflow:auto;">
+ 
+ 		<table border="1">
+ 			<tr>
+ 				<th>Product ID</th>
+ 				<th>Item Name</th>
+ 				<th>Quantity</th>
+ 				<th>Price</th>
+ 			</tr>
+ 		</table>
+ 
+ 
+ 		<table border="0">
+ 			<tr style="font-size:13px">
+ 				{% for some in results %}
+
+ 					<td><div id="IN" align:'left'>
+ 						</div> {{some[0]}}</td>
+ 					<td><div id="I">
+ 					</div> {{some[1]}} </td>
+ 
+ 					<td><div id='Q'>
+ 						<div> {{some[2]}} </div>
+ 					</div></td>
+ 					<td><div id="P">
+ 						<div> {{some[3]}} </div>
+ 					</div></td>
+ 
+ 				{% endfor %}
+ 			</tr>
+ 		</table>
+ 
+ 	</div>
+ 
+ 
+ 	<div align="right">
+ 	<br>
+ 	<form action = "/cart" method = "post">
+   		Product ID:
+   		<input type="text" name="ItemNumber"><br></input>
+   		Quantity:
+   		<input type="text" name="quantity"><br></input>
+   		<input type="submit" value="Add">
+ 	</form>
+ 
+ 	<br>
+ 	<br>
+ 
+ 	<button type="button">Check Out</button><br>
```

Comment:
We can actually put a `<a>` tag here in this button so it will take you to `/checkout`